SonarScanner.Shim/PropertiesFileGenerator.cs

```diff
@@ -28,16 +28,30 @@
 
 namespace SonarScanner.Shim
```

Comment:
I would love to convert this file to generate a list with property-value pairs, but it seems too much work